<launch>    
	<!-- Arguments -->
	<arg name="filename" default="$(find occupancy_map_visualizer)/samples/quadtree.ot2" />

    <!-- Parameters -->
    <param name="fixed_frame_id"   value="map" />
    <param name="quadmap_filename" value="$(arg filename)" />

    <!-- Run Rviz -->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find occupancy_map_visualizer)/rviz/quadmap.rviz" />

    <!-- Quadtree Visualization -->
    <node pkg="occupancy_map_visualizer"	type="quadmap_visualizer_node"	name="quadmap_visualizer_node"	output="screen"/>
</launch>
